nwamugo/politico-react
# 166395672-feature(profile): user can view profile

#### What does this PR do?
This PR allows the user to view his profile

#### Description of Task to be completed?
After logging in, user can see his profile details

#### How should this be manually tested?
Go to https://nwamugo-politico.netlify.com/
Sign up or login
You will see your profile page afterwards

#### What are the relevant pivotal tracker stories?(if applicable)
#166395672 User can view profile
